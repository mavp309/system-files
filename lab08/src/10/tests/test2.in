1239871 123
