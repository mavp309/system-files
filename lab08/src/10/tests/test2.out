123 1239871
